final_task/calc/basic_concepts.py

```diff
@@ -0,0 +1,71 @@
+import math
+import operator
+
+
+def operator_priority(symbol):
```

Comment:
Consider renaming this function into `get_operator_priority` since it returns priority for particular input symbol